config/routes.rb

```diff
@@ -29,6 +26,10 @@
     end
   end
 
+  resources :orders, only: :show
+  resources :products, only: [:show, :index]
+
+  get 'home/index' , to: 'home#index'
   get 'cart', to: 'cart#index'
   get 'cart/address', to: 'cart#address'
```

Comment:
The home index one was removed, but the other two resources were needed.  The site would error out with out the resources.